pkg/web/static/js/main.js

```diff
@@ -68,17 +68,20 @@ const updateDatetimes = () => {
 document.body.addEventListener('htmx:afterSettle', updateDatetimes);
 
 // Disable submit button after form submission and enable it after a request.
-const submitBtn = document.querySelector('.submit-btn');
-const submitBtnText = document.querySelector('.submit-btn-text');
-
 function disableSubmitBtn() {
+  const submitBtn = document.querySelector('.submit-btn');
```

Comment:
The change was made to ensure this function works for the accept and repair submit buttons.